### PR TITLE
feat(@spartacus/core): remove clearmiscsdata action from user's meta reducer 

### LINKED ISSUE #1190: Remove ClearMiscsData action from user's meta reducer
Refactor: `projects/core/src/user/store/reducers/index.ts`

### PR DESCRIPTION
**Pull request by znikola**
closes #1190

### CODE REVIEW

File: projects/core/src/user/store/actions/regions.action.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { Action } from '@ngrx/store';
+
 import { Region } from '../../../occ/occ-models/index';
 
 export const LOAD_REGIONS = '[User] Load Regions';
 export const LOAD_REGIONS_SUCCESS = '[User] Load Regions Success';
 export const LOAD_REGIONS_FAIL = '[User] Load Regions Fail';
+export const RESET_REGIONS = '[User] Reset Regions';
```
Comment (dunqan) [inline projects/core/src/user/store/actions/regions.action.ts]: Can we remove it?

File: projects/core/src/user/store/actions/titles.action.ts [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import { Action } from '@ngrx/store';
+
 import { Title } from '../../../occ/occ-models';
 
 export const LOAD_TITLES = '[User] Load Tiltes';
 export const LOAD_TITLES_FAIL = '[User] Load Titles Fail';
 export const LOAD_TITLES_SUCCESS = '[User] Load Titles Success';
+export const RESET_TITLES = '[User] Reset Tiltes';
```
Comment (dunqan) [inline projects/core/src/user/store/actions/titles.action.ts]: Can we remove it?

File: projects/core/src/user/store/reducers/delivery-countries.reducer.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ export const initialState: DeliveryCountriesState = {
 
 export function reducer(
   state = initialState,
-  action: fromAction.DeliveryCountriesAction | fromAction.MiscsDataAction
+  action: fromAction.DeliveryCountriesAction
```
Comment (dunqan) [inline projects/core/src/user/store/reducers/delivery-countries.reducer.ts]: What about `MiscDataAction`? Wasn't it used before to clear countries state?

File: projects/core/src/user/store/reducers/regions.reducer.ts [hunk 4]
```diff
@@ -8,7 +8,7 @@ export const initialState: RegionsState = {
 
 export function reducer(
   state = initialState,
-  action: fromAction.RegionsAction | fromAction.ClearMiscsData
+  action: fromAction.RegionsAction
```
Comment (dunqan) [inline projects/core/src/user/store/reducers/regions.reducer.ts]: What about ClearMiscData action?

File: projects/core/src/user/store/reducers/titles.reducer.ts [hunk 5]
```diff
@@ -8,7 +8,7 @@ export const initialState: TitlesState = {
 
 export function reducer(
   state = initialState,
-  action: fromAction.TitlesAction | fromAction.MiscsDataAction
+  action: fromAction.TitlesAction
```
Comment (dunqan) [inline projects/core/src/user/store/reducers/titles.reducer.ts]: What about MiscDataAction?

File: projects/core/src/user/store/actions/regions.action.ts (hunk 1 above)
Comment (znikola) [reply projects/core/src/user/store/actions/regions.action.ts]: Good catch

File: projects/core/src/user/store/actions/titles.action.ts (hunk 2 above)
Comment (znikola) [reply projects/core/src/user/store/actions/titles.action.ts]: Good catch

File: projects/core/src/user/store/reducers/delivery-countries.reducer.ts (hunk 3 above)
Comment (znikola) [reply projects/core/src/user/store/reducers/delivery-countries.reducer.ts]: Good catch

File: projects/core/src/user/store/reducers/regions.reducer.ts (hunk 4 above)
Comment (znikola) [reply projects/core/src/user/store/reducers/regions.reducer.ts]: Good catch

File: projects/core/src/user/store/reducers/titles.reducer.ts (hunk 5 above)
Comment (znikola) [reply projects/core/src/user/store/reducers/titles.reducer.ts]: Good catch

**Pull request merged**